resources/data/contributors.json

```diff
@@ -1060,6 +1060,40 @@
           43.7005580
         ]
       }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "Name": "Dimas Ciputra",
+        "Committer": "Yes",
```

Comment:
I asked them to add themselves. True they don't contribute to core but they make a lot of contributions to the project as a whole. Many contributors are pretty much invisible to the project and the contributors JSON is a great way for us to showcase how rich and geographically diverse our community is.

Maybe we can add a contribution topics field to distinguish which areas folks contribute to?

Reply:
That's nice to have representation for all contributors.

> Maybe we can add a contribution topics field to distinguish which areas folks contribute to?

sounds good. like ?
"Contribution topics" : [ "Code", "Documentation", "QGIS-Django", "Translation", ... ]
But you can be a committer (=push rights) in one area and not in another one. hum...

Looking at doc/developersmap.html, it would need some adaptations since it uses the Committer = Yes to distinguish from "Core committer" and "Core contributor"
```
        layer.bindPopup(nameString + "<br>" +
                        (feature.properties.Committer == "Yes" ?
                        "Core committer" : "Core contributor" ));
```
I assume that "Core" means to the code itself. Should we do such distinction in that map ? I don't know

Anyway just wanted to point things from a consistency point of view of our data, not creating fences between contributors.

Reply:
Hi @rouault 

> Anyway just wanted to point things from a consistency point of view of our data, not creating fences between contributors.


Right - just a note that I am not creating precedence here - the list already contains people who are not coders who have contributed to QGIS in other ways.

![image](https://user-images.githubusercontent.com/178003/153090718-3c69335e-a41e-415b-aa21-4bff6fb77ca5.png)

I was thinking that we could do a bit of a broader call to encourage people who contribute to the project to add themselves so that we can really see the depth and breadth of the team that makes QGIS happen, and try to reduce the innate bias of the map towards core developers (who are typically more aware of the contributors.json document). Historical note: This was also the reason I originally called this file 'contributors' rather than 'developers' back in 2013, though I did a crap job of soliciting the community of contributors to add themselves to it.

Reply:
> the list already contains people who are not coders who have contributed to QGIS in other ways.

I'm totally in favour of this!

I do think we may need some guidelines about what makes inclusion acceptable. Some easy qualifications could be:
- have committed some code to **any** of the qgis/* repos
- are a community voting member
- others :wink:

Reply:
Maybe:

- have contributed financially to QGIS development through a direct contribution or crowd funding contribution?

Reply:
> Right - just a note that I am not creating precedence here

ok, that's fine then. The "wooh there's a lot of people from everywhere on this map" effect is probably more important than having an hyper structured "database" :-)